addon/components/input-validator.js

```diff
@@ -47,14 +53,16 @@ export default Component.extend({
         return set(this, 'hasFocusedOut', true);
     },
 
-    didRender() {
+    didInsertElement() {
         this._super(...arguments);
-        const input = this.element.querySelector('input, select, textarea');
-        const label = this.element.querySelector('label.input-validator-label');
-        if(input && label) {
-            label.setAttribute('for', input.id);
-        }
+        scheduleOnce('afterRender', this, () => {
```

Comment:
`scheduleOnce` is not guaranteed to run only once if you give it an anonymous function like this. To be safe, I'd recommend either changing this to just `schedule()` (if you dont care if it could run multiple time), or make the anonymous function a class method, and call it like `scheduleOnce('afterRender', this, 'setForAttribute')`